firmware/common2015/utils/logger/logger.hpp

```diff
@@ -24,6 +25,8 @@
 #define LOG(lvl, ...) \
     log(lvl, __BASE_FILE_NAME__, __LINE__, __func__, __VA_ARGS__)
 
+#define S_LOG(lvl) LogHelper(lvl, __BASE_FILE_NAME__, __LINE__, __func__)
```

Comment:
Also, the `std::endl` isn't needed since the `log()` function already appends one, right?